validation-test/Sema/type_checker_perf/slow/rdar54580427.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: %scale-test --begin 1 --end 10 --step 1 --select NumLeafScopes %s
+// REQUIRES: OS=macosx
+// REQUIRES: asserts
+// XFAIL: *
```

Comment:
Instead of XFAIL here you can use `--invert-result` to verify that it's indeed exponential in the number of scopes.

Reply:
I don't follow how that verifies that it is indeed exponential? If it were quadratic (say), then using `--invert-result` would give a `sqrt` which would still pass, right?

Reply:
I think it verifies that it's not linear instead which is good enough. So once it becomes linear it would start failing.